integration-test.sh

```diff
@@ -25,3 +26,4 @@ echo "Testing in-place upgrade and customizations"
 ./install.sh --minimize-downtime
 _integration-test/run.sh
 _integration-test/ensure-customizations-work.sh
+_integration-test/ensure-backup-restore-works.sh
```

Comment:
A little goofy that we have to call this twice, but I guess that's where we're at and we clean up the test suite under https://github.com/getsentry/team-ospo/issues/91.